loop_utils.c

```diff
@@ -0,0 +1,80 @@
+#include "./dukluv/src/duv.h"
```

Comment:
@WorkForPizza - this file was adapted from https://github.com/sh3nan1gans/dukluv/blob/master/src/main.c